ocaml/idl/datamodel.ml

```diff
@@ -2372,7 +2374,7 @@ module VDI = struct
       ~allowed_roles:_R_VM_ADMIN
       ()
 
-  let db_introduce = { pool_introduce with msg_name = "db_introduce"; msg_hide_from_docs = false }
+  let db_introduce = { pool_introduce with msg_name = "db_introduce"; msg_hide_from_docs = true }
 
   let db_forget = call
```

Comment:
Should we add `hide_from_docs` to this too, it was listed on the ticket?

Reply:
Well spotted!